webpack.dev.js

```diff
@@ -15,5 +15,9 @@ module.exports = merge(common, {
   plugins: [new webpack.HotModuleReplacementPlugin()],
   optimization: {
     usedExports: true,
+    splitChunks: {
```

Comment:
What does this do?

Reply:
Oh, we were trying to optimize compile time during dev, unfortunately did not work. I will get rid of this.